### PR TITLE
Media

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: src/Bundle/Admin/Resources/views/form_theme.html.twig [hunk 1]
```diff
@@ -208,3 +208,77 @@
...
+    <div class="ui grid">
+        <div class="row">
+            {% if value %}
+            {% set description = psi_describe(value) %}
+
+            <div class="eight wide column">
+                {% if description.has('std.image') %}
+                    <img class="ui inline big spaced image" src="{{ description.get('std.image').value | imagine_filter('preview') }}" />
+                {% endif %}
+            </div>
```
Comment (dantleech) [inline src/Bundle/Admin/Resources/views/form_theme.html.twig]: Dentation

**Comment by maurosergiosilva** [comment]
You´re doing a Great Job Dantleech :)!

File: src/Bundle/Article/Document/Page.php [hunk 2]
```diff
@@ -43,6 +44,14 @@ class Page implements RouteReferrersReadInterface
     private $publishPeriod;
 
     /**
+     * @ContentType\Property(type="object_reference", options={
+     *     "class": "Trog\Bundle\Media\Document\File",
+     *     "browser": "selector"
+     * }, role="image")
+     */
```
Comment (dantleech) [inline src/Bundle/Article/Document/Page.php:51]: Filters / only allow selection by specified mime type. Description validator? (e.g. mimeType=image/*)

**Comment by dantleech** [comment]
Thanks

**Pull request merged**